### DISCUSSION TITLE
List of dependents when repo contains more than one package

### DISCUSSION THREAD

**Discussion by omri374** [community/community#106737] [Repositories]
### Select Topic Area

Bug

### Body

Hi,
We have a repo which contains 3 different Python packages: https://www.github.com/microsoft/presidio. Two out of the three packages are downloaded more than 150K times a month.

When looking in the dependents section, I can only see dependents of one package out of the three:
<img width="957" alt="image" src="https://github.com/community/community/assets/3776619/c128a194-410a-4480-94fb-97bc4ccc7992">

Is there a way to contain the list of dependents for all packages in the repo?

**Comment by sophies927** [comment]
I have the same question. My team's repo contains several packages across multiple different package managers, and only some are displayed in the "Packages" dropdown menu within the "Dependents" page. How are the packages determined? Is it possible to somehow link the dependent packages to my repo so I can see dependents for all our packages?
